secboot/secboot_tpm.go

```diff
@@ -354,10 +351,10 @@ func unlockEncryptedPartitionWithKey(name, device string, key []byte) error {
 	return err
 }
 
-// SealKey provisions the TPM and seals a partition encryption key according to the
+// SealKey provisions the TPM and seals the encryption keys according to the
 // specified parameters. If the TPM is already provisioned, or a sealed key already
 // exists, SealKey will fail and return an error.
-func SealKey(key EncryptionKey, params *SealKeyParams) error {
+func SealKey(keys []SealKeyRequest, params *SealKeyParams) error {
```

Comment:
SealKeys?

Reply:
SealKeys is a better name, will replace. I just copied the name used in snapcore/secboot to get a quicker start, but I don't think we should keep that name.

Reply:
Yeah, had the same question

Reply:
Renamed to SealKeys & ResealKeys